talend-component-maven-plugin/src/it/web/.jenkins/Jenkinsfile

```diff
@@ -118,8 +130,19 @@ pipeline {
    * environment
    */
   environment {
-    MAVEN_SETTINGS = "${SETTINGS_PATH}"
-    DECRYPTER_ARG = "-Dtalend.maven.decrypter.m2.location=${env.WORKSPACE}/.jenkins/"
+    APP_ID = '579232'
+    TALEND_REGISTRY = "artifactory.datapwn.com"
+    TESTCONTAINERS_HUB_IMAGE_NAME_PREFIX = "artifactory.datapwn.com/docker-io-remote/"
+
+    String workspace = "${WORKSPACE}"
+    build_user = "${BUILD_USER}"
+
+    scripts_path = "${workspace}/talend-component-maven-plugin/src/it/web/.jenkins/scripts"
+    tests_path = "${workspace}/talend-component-maven-plugin/src/it/web/test"
+    String settings_path = "${workspace}/talend-component-maven-plugin/src/it/web/.jenkins/settings.xml"
+
+    MAVEN_SETTINGS = "${settings_path}"
```

Comment:
Can we use directly the line above here to avoid confusion with env variable of the same name ?

Reply:
Clearly